webpack/assets/javascripts/react_app/redux/reducers/statistics/index.js

```diff
@@ -10,20 +10,20 @@ const initialState = Immutable({
 });
 
 export default (state = initialState, action) => {
-  const { payload } = action;
+  const { payload, response } = action;
 
   switch (action.type) {
     case STATISTICS_DATA_REQUEST:
       return state.setIn(['charts', payload.id], payload);
     case STATISTICS_DATA_SUCCESS:
       return state.setIn(['charts', payload.id], {
```

Comment:
should we be using the id from the payload or from the response (here and elsewhere)?

Reply:
imo we should use it from the payload, not sure if every response returns the id too.